diff --git a/PCbuild/_ssl.vcxproj b/PCbuild/_ssl.vcxproj
index d75ebd656a..c19d20f5b5 100644
--- a/PCbuild/_ssl.vcxproj
+++ b/PCbuild/_ssl.vcxproj
@@ -86,7 +86,7 @@
       <Project>{10615b24-73bf-4efa-93aa-236916321317}</Project>
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
-    <ProjectReference Include="_socket.vcxproj">
+    <ProjectReference Condition="false" Include="_socket.vcxproj">
       <Project>{86937f53-c189-40ef-8ce8-8759d8e7d480}</Project>
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
diff --git a/PCbuild/pcbuild.proj b/PCbuild/pcbuild.proj
index f7890969c8..8258b486f2 100644
--- a/PCbuild/pcbuild.proj
+++ b/PCbuild/pcbuild.proj
@@ -43,18 +43,18 @@
       <BuildInParallel>false</BuildInParallel>
     </Projects>
     <!-- python3.dll -->
-    <Projects Include="python3dll.vcxproj" />
+    <Projects Include="python3dll.vcxproj" Condition="false" />
     <!-- py[w].exe -->
-    <Projects Include="pylauncher.vcxproj;pywlauncher.vcxproj" />
+    <Projects Include="pylauncher.vcxproj;pywlauncher.vcxproj" Condition="false" />
     <!-- pyshellext.dll -->
-    <Projects Include="pyshellext.vcxproj" />
+    <Projects Include="pyshellext.vcxproj" Condition="false" />
     <!-- Extension modules -->
     <ExtensionModules Include="_asyncio;_ctypes;_decimal;_distutils_findvs;_elementtree;_msi;_multiprocessing;_overlapped;pyexpat;select;unicodedata;winsound" />
     <!-- Extension modules that require external sources -->
     <ExternalModules Include="_bz2;_lzma;_sqlite3" />
-    <!-- _ssl will build _socket as well, which may cause conflicts in parallel builds -->
-    <ExtensionModules Include="_socket" Condition="!$(IncludeSSL) or !$(IncludeExternals)" />
-    <ExternalModules Include="_ssl;_hashlib" Condition="$(IncludeSSL)" />
+    <!-- _ssl will NOT build _socket as well -->
+    <ExtensionModules Include="_socket" Condition="true" />
+    <ExternalModules Include="_ssl;_hashlib" Condition="true" />
     <ExternalModules Include="_tkinter;tix" Condition="$(IncludeTkinter)" />
     <ExtensionModules Include="@(ExternalModules->'%(Identity)')" Condition="$(IncludeExternals)" />
     <Projects Include="@(ExtensionModules->'%(Identity).vcxproj')" Condition="$(IncludeExtensions)" />
@@ -117,4 +117,4 @@
 
   <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
   <Target Name="RebuildAll" DependsOnTargets="CleanAll;Build" />
-</Project>
\ No newline at end of file
+</Project>
